shared/templates/template_IGNITION_service_disabled

```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+# reboot = true
+# strategy = disable
+# complexity = low
+# disruption = medium
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+      - name: {{{ DAEMONNAME }}}.service
+        enabled: false 
+{{%- if MASK_SERVICE %}}
+        mask: true
+{{%- endif %}}
+      - name: {{{ DAEMONNAME }}}.socket
```

Comment:
What will happen if the socket file doesn't exist?

Reply:
AFAIU, it shouldn't be a problem as it should just remove the file in the targets directory. Or if "masked" is enabled, it should just point it to /dev/null